pronto/api/signature.py

```diff
@@ -647,3 +650,26 @@ def digest(self, obj: dict):
             key = "similarity"
 
         return i, j, -obj["residues"][key], -obj[key]
+
+
+def check_if_ncbifam_amr(method_ac):
```

Comment:
Type hinting and better variable name:

```suggestion
def check_if_ncbifam_amr(accession: str) -> bool:
```